default.nix

```diff
@@ -0,0 +1,12 @@
+let
+  sources = import ./sources.nix;
+  pkgs = import sources.nixpkgs {
+    config = {};
+    # TODO: overlay doesn't work yet
+    # overlays = [ (import ./overlay.nix) ];
```

Comment:
@flokli or @arianvp I need help here. This syntax doesn't quite work, and/or maybe the syntax inside overlay.nix doesn't quite work.

To test things I ran `nix-build default.nix` with and without commenting this line.